cli/functionary/utils.py

```diff
@@ -45,6 +47,21 @@ def flatten(results, object_fields):
     return new_results
 
 
+def _fix_datetime_display(value):
+    """
+    Helper function for format_results to remove milliseconds from datetime
+
+    Args:
+        value: string representing datetime value
+    Returns:
+        value as a string representing datetime value without milliseconds
+    """
+    value = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S.%f%z").replace(
```

Comment:
What's the reason replacing the microseconds?

Reply:
Was originally to remove the microseconds from the datetime object, but you're right that we don't need it now